addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -13,16 +13,38 @@
 
     @include print-variable('priority-palette-prefix', $o-palette-priority-prefix);
 
-    // Need the colors and name of each color palette
+    // Need the colors, name, menu background color and footer background color of each color palette
     $palette-names: ();
+    $default-presets: ();
+    $bg-attrs: 'menu', 'footer';
+    @each $attr in $bg-attrs {
+        $default-presets: map-merge($default-presets, ($attr: map-get($o-base-color-palette, $attr)))
+    }
+
     @each $palette-name, $palette in $o-color-palettes {
         @if $palette-name != 'user-palette' {
             @each $key, $value in $palette {
                 @include print-variable('o-palette-#{$palette-name}-#{$key}', $value);
             }
             $palette-names: append($palette-names, $palette-name);
+
+            @each $attr-name, $default-preset-idx in $default-presets {
+                $preset-idx: o-safe-get($palette, $attr-name, $default-preset-idx);
+                $preset: nth($o-color-combinations, $preset-idx);
+                $bg: o-safe-get($palette, 'o-cc#{$preset-idx}-bg', map-get($preset, 'bg'));
+                $bg: o-safe-get($palette, $bg, $bg);
```

Comment:
What's this one, I am not sure

Reply:
`$bg `is either a hex color (basically when it comes from `o-ccX-bg`) or a color 'alias' like `o-color-2` when it comes from the color combination which is wrongly named `$preset` here (what I describe here is what is done in the line just above). This line allows to transform the color alias to an actual hex color. And if the color is already a hex color we fallback on it.

Reply:
Maybe worth being explicit with
```scss
@if type-of($bg) != color {
    $bg: map-get($palette, $bg);
}
```